src/Campaigns/Migrations/MigrateFormsToCampaignForms.php

```diff
@@ -89,7 +89,7 @@ protected function getFormData(): array
      */
     protected function getUpgradedFormData(): array
     {
-        return DB::table('posts', 'forms')
+        return DB::table('posts', 'forms')->distinct()
```

Comment:
We can remove this `distinct`

Reply:
Resolved in 9ac7b978746a1b487ffb4ba47c472147d538d6a1